source/includes/quick-start/atlas-setup.rst

```diff
@@ -4,7 +4,7 @@ Set up a Free Tier Cluster in Atlas
 After setting up your Java project dependencies, create a MongoDB cluster
 where you can store and manage your data. Complete the
 :atlas:`Get Started with Atlas </getting-started>` guide to set up a new
-Atlas account, free tier MongoDB cluster, load datasets, and
+Atlas account, build a free tier MongoDB cluster, load datasets, and
```

Comment:
I think "create and launch" are more appropriate. The user doesn't perform any builds.

Reply:
I agree "create and launch" is more appropriate here.